adal/src/telemetry/java/com/microsoft/aad/adal/TelemetryUtils.java

```diff
@@ -24,16 +24,36 @@
 package com.microsoft.aad.adal;
 
 import java.io.Serializable;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
+import static com.microsoft.aad.adal.EventStrings.*;
```

Comment:
i know you are trying to avoid doing EventStrings.LOGIN_HINT et al, which makes sense here, but we do not want to set precedence for importing .* in our code, can you either remove this and change the fields below or import specific fields

Reply:
Resolved 👍